README.non-docker-test-environments.md

```diff
@@ -32,6 +32,25 @@ For more information about installing Titan, please go the website above.
 
 * But if all this sounds too complicated, or if you prefer not to install Titan: 
   * You can uncomment the section in `src/test/resources/docker/docker-compose.yml` which corresponds to the platform you are interested in
+    * Be aware, we use same ports for different versions of same platform in our CI/CD. Change host port for the container if you want to run two versions simultaneously. 
+    And change port in connection url in `harness-config.yml` accordingly.
   * Connect to the database 
-  * Copy and paste the SQL statements from `src/test/resources/docker{DnName}-init.sql`. 
+  * Copy and paste the SQL statements from `src/test/resources/docker{DbName}-init.sql`. 
   * And now you can run tests against this platform
+
+### Adding new Titan managed platform by HSQLDB 2.5 example
+##### Remark - this is not comprehensive guide that can teach all you need about Titan, just an example how we do it. Read [Titan docs](https://titan-data.io/docs) to get idea how things work.
+* Find container in docker hub that suits Titan. Titan required at least one volume declared in docker image; 
+* Run this image with Docker to see if it's working properly, figure out correct container configuration, if you can connect to DB, run some sql commands, etc.;
+* Make sure your Titan is up and running
+* Run controlled container like this `titan run -e HSQLDB_USER=lbuser -e HSQLDB_PASSWORD=LiquibasePass1 -e HSQLDB_DATABASE_ALIAS=lbcat -n hsqldb-2.5  mitchtalmadge/hsqldb:2.5.0`
+  * `-e` is env variable for container, `-n` is name. 
+  * To remap ports add next flags `-P -- -p 9002:9001`. `-P` disables the default port mapping `--` allows passing context specific arguments (context is docker in this case).
+* Connect to DB and run init script to populate it with test data. Init script for HSQLDB is in `src/test/resources/docker/hsqldb-init.sql`.
+* Commit changes `titan commit -m "data loaded" hsqldb-2.5`.
+* Make sure you have write access to folder in S3 bucket and your AWS cli credentials are configured. For our case it's `test-harness-titan-configs` bucket with `hsqldb-2.5` folder.
+* Add remote, for our case it's `titan remote add s3://test-harness-titan-configs/hsqldb-2.5 hsqldb-2.5`.
+* Run `titan log hsqldb-2.5` to get commit id
+* Run `titan push -c c3be54d46da94dcd80c3fdbcae5a4494 hsqldb-2.5`. c3be54d46da94dcd80c3fdbcae5a4494 is commit hash provided by `log` command.
```

Comment:
If you push without `-c c3be54d46da94dcd80c3fdbcae5a4494`, it will push the last commit.